packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.css

```diff
@@ -44,12 +44,8 @@ limitations under the License.
   text-align: right;
 }
 
-.KeyValueTable--copyIcon {
-  visibility: hidden;
-}
-
-.KeyValueTable--row:hover > .KeyValueTable--copyColumn > .KeyValueTable--copyIcon {
-  visibility: unset;
+.KeyValueTable--row:not(:hover) > .KeyValueTable--copyColumn > .KeyValueTable--copyIcon {
```

Comment:
Is the use of `:not(:hover)` to avoid defining two styles? :+1:

Reply:
yup! seemed simpler